src/components/accordion/accordion.stories.mdx

```diff
@@ -0,0 +1,45 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+import { AccordionContainerExample, AccordionSectionExample, Regular } from './accordion.stories';
+import { Accordion, AccordionSection } from './accordion';
+
+<Meta
+    title="Components/Accordion"
+    component={{ Accordion }}
+    subcomponents={{AccordionSection}} />
+
+# Accordion
+An accordion is a vertically stacked group of collapsible sections. Our accordion is based on the [Accordion](https://reach.tech/accordion/) from ReachUI. We have baked in many of our configuration and design decisions which makes it easier to use. 
+
+**Some notable things:**
+
++ Every accordion receives internally the toggles `multiple` & `collapsible` which you can't remove. 
++ `AccordionContent` is the only child component to define your content.
++ You can't define your own `AccordionButton`, `AccordionPanel` and such, everything is predefined and bundled in the `AccordionContent` component.
+
+<Canvas>
+  <Story story={Regular}/>
+</Canvas>
+
+
+## Component API
+
+<ArgsTable of={Accordion} />
```

Comment:
Because I defined a subcomponent of interest in the meta data (`AccordionSection`) we get a second tab.

![Screen Shot 2021-11-24 at 10 47 20](https://user-images.githubusercontent.com/1701755/143214578-a44e91a1-53f9-42ec-a6ea-0a6fccf9ecfe.png)

The behavior is not expected actually. I thought I pass in the story and I get the story props but it always shows me the two tabs, regardless of the story I pass in.